src/Components/Dynamic/DynamicCard.tsx

```diff
@@ -1,7 +1,9 @@
-import { linkService } from "../../Services/link.service.ts";
-import { Restaurant, Dish } from "../../Assets/data.ts";
+import { linkService } from "../../Services/link.service";
+import { Restaurant, Dish } from "../../Assets/data";
 import { Link } from "react-router-dom";
 import { Rating } from "@mui/material";
+import { toggleModal } from "../../store/actions/modal.actions";
+import { useDispatch } from "react-redux";
 
 interface DynamicCardProps {
```

Comment:
maybe add url prefix prop
and then no need is restaurant
or just add optional prop is restaurant.